File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int length = nums.length;
        int[] result = new int[length * 2];
        for(int i=0; i<length; i++){
            result[i] = nums[i];
            result[i+length] = nums[i];
        }
        return result;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        freq = 1
        element = nums[0]
        for num in nums:
            if element == num:
                freq += 1
            else:
                freq -= 1
                if freq <= 0:
                    freq = 1
                    element = num
        return element 